### PR TITLE
ENH: wip, convert from Decapodes to DecaExpr

### PR DESCRIPTION
**Pull request by jpfairbanks**

### CODE REVIEW

**Comment by lukem12345** [comment]
What's the status on this? Is there code to solve this that will live in syntactic models?

**Comment by lukem12345** [comment]
It would be nice to have a feature extension on top of the conversion to a DecaExpr further towards something that looks like the LaTeX-like body we started with.

**Comment by lukem12345** [comment]
Tests should fail due to the afore-mentioned dependency on development branch of ACSets.

Owen's intertype branch should be merged before this.

**Comment by lukem12345** [comment]
Note that composed operators are now expanded in Decapode deconstruction.

**Comment by jpfairbanks** [comment]
@lukem12345 can you approve and merge this? Since I made the PR, I can't approve it.

File: Project.toml [hunk 1]
```diff
@@ -4,6 +4,7 @@ authors = ["James Fairbanks", "Andrew Baas", "Evan Patterson", "Luke Morris", "G
 version = "0.5.0"
 
 [deps]
+ACSets = "227ef7b5-1206-438b-ac65-934d6da304b8"
```
Comment (lukem12345) [inline Project.toml:7]: I don't see a compat for ACSets, but CompatHelper can catch this if it's not already in main.

**Review by lukem12345 — APPROVED** [review]

**Pull request merged**